tests/utils/test_local_scheduler_remote_worker_local_postgres.py

```diff
@@ -0,0 +1,227 @@
+import subprocess
```

Comment:
we use a file-level docstring as well. you can find examples in other gtsfm files. 
Also this file would be better located in the scripts folder instead of the tests folder. the tests folder is for unit tests. https://github.com/borglab/gtsfm/tree/master/scripts